analyser_merge_milestone_FR_metropole.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyrights Frédéric Rodrigo 2014-2020                                 ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from collections import OrderedDict
+from .Analyser_Merge import Analyser_Merge, Source, CSV, Load, Mapping, Select, Generate
+
+class Analyser_Merge_Milestone_FR_metropole(Analyser_Merge):
+    def __init__(self, config, logger = None):
+        Analyser_Merge.__init__(self, config, logger)
+
+        # fix item 
+        self.def_class_missing_official(item = 8130, id = 41, level = 3, tags = ['merge', 'milestone'],
+            title = T_('Milestone not integrated'))
+        self.def_class_possible_merge(item = 8131, id = 43, level = 3, tags = ['merge', 'milestone'],
+            title = T_('Milestone integration suggestion'))
+        self.def_class_update_official(item = 8132, id = 44, level = 3, tags = ['merge', 'milestone'],
+            title = T_('Milestone update'))
```

Comment:
```suggestion
        self.def_class_missing_official(item = 8130, id = 41, level = 3, tags = ['merge', 'highway'],
            title = T_('Milestone not integrated'))
        self.def_class_possible_merge(item = 8131, id = 43, level = 3, tags = ['merge', 'highway'],
            title = T_('Milestone integration suggestion'))
        self.def_class_update_official(item = 8132, id = 44, level = 3, tags = ['merge', 'highway'],
            title = T_('Milestone update'))
```

Reply:
item 8130, 8131 and 8132 is for Carpool parking
must be changed to ?

Reply:
Yes, I do not look at it, yet.